grafana-plugin/src/models/alert_receive_channel/alert_receive_channel.ts

```diff
@@ -91,11 +91,14 @@ export class AlertReceiveChannelStore extends BaseStore {
   async loadItem(id: AlertReceiveChannel['id'], skipErrorHandling = false): Promise<AlertReceiveChannel> {
     const alertReceiveChannel = await this.getById(id, skipErrorHandling);
 
+    // @ts-ignore
     this.items = {
       ...this.items,
-      [id]: alertReceiveChannel,
+      [id]: omit(alertReceiveChannel, 'heartbeat'),
```

Comment:
can we store `heartbeat` on the objects within `items? why do we need `alertReceiveChannelToHeartbeat`? I believe a heartbeat _always_ belongs to a single alert receive channel.

Reply:
in my mind heartbeat is a separate entity (it even has it's own id and ideally it should have separate endpoint), that's why we have 'models/heartbeat', we need `alertReceiveChannelToHeartbeat` to save appropriate heartbeat Id for each alert receive channel, it's one to one mapping. Yes we can store heartbeat id  just in alert receive channel object, but I did not want to pollute  alert receive channel object